src/libraries/Microsoft.Extensions.FileProviders.Physical/tests/PhysicalFileProviderTests.cs

```diff
@@ -1512,6 +1512,66 @@ public void UsePollingFileWatcher_FileWatcherNotNull_ReturnsFalse()
             }
         }
 
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
+        public async Task UsePollingFileWatcher_UseActivePolling_HasChanged(bool useWildcard)
+        {
+            // Arrange
+            using var root = new DisposableFileSystem();
+            string fileName = Path.GetRandomFileName();
+            string filePath = Path.Combine(root.RootPath, fileName);
+            File.WriteAllText(filePath, "v1.1");
+
+            using var provider = new PhysicalFileProvider(root.RootPath) { UsePollingFileWatcher = true, UseActivePolling = true };
+            IChangeToken token = provider.Watch(useWildcard ? "*" : fileName);
+            Assert.False(token.HasChanged);
+
+            // Act
+            await Task.Delay(200); // Wait a bit before writing again, see https://github.com/dotnet/runtime/issues/55951.
+            File.WriteAllText(filePath, "v1.2");
+            await Task.Delay(GetTokenPollingInterval(token));
+
+            // Assert
```

Comment:
what is the value of having the AAA comments?

Reply:
It helps to comprehend the tests easier. Also, the pattern was already there so I just continued with it.